tests/PSDocs.Tests/PSDocs.Tests.csproj

```
@@ -14,7 +14,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
     <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.0.3" />
```

tests/PSDocs.Tests/PSDocs.Tests.csproj
```suggestion
     <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.0.3" />
```

---

> tests/PSDocs.Tests/PSDocs.Tests.csproj
> ```suggestion
>      <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.0.3" />
> ```
>